src/main/sensors/acceleration.c

```diff
@@ -78,9 +78,10 @@ void accUpdate(timeUs_t currentTimeUs)
 
     applyAccelerationTrims(accelerationRuntime.accelerationTrims);
 
-    for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
-        const int16_t val = acc.accADC[axis];
-        acc.accADC[axis] = accelerationRuntime.accLpfCutHz ? pt2FilterApply(&accelerationRuntime.accFilter[axis], val) : val;
+    if (accelerationRuntime.accLpfCutHz) {
+        for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
+            acc.accADC.v[axis] = pt2FilterApply(&accelerationRuntime.accFilter[axis], acc.accADC.v[axis]);
```

Comment:
Eventually, it may be useful to implement vector version of filters, it will repeat a lot in code. And filter coefficients are the same in most cases, so it will allow some optimization.